schema/application/resolvers/Mutation/saveChecklist.ts

```diff
@@ -757,6 +757,30 @@ async function saveChecklistResults(
                           AND
                           workresultisrequired != inputs.required
                   `,
+                  i.result.order
+                    ? tx`
+                      WITH inputs (order) AS (
+                            VALUES (
+                            ${i.result.order ?? null}::integer
+                            )
+                      )
+                      UPDATE public.workresult
+                      SET 
+                          workresultorder = inputs.order,
+                          workresultmodifieddate = now()
+                      FROM inputs
+                      WHERE
+                          id = ${decodeGlobalId(i.result.id).id}
+                          AND
+                          workresultorder != inputs.order
```

Comment:
This needs a little more because of (postgre)sql's NULL semantics;
```
AND (
    workresultorder IS null
    OR workresultorder != inputs.order
)
```